docs/content/en/docs/install/cert-manager.md

```diff
@@ -27,73 +27,45 @@ The steps are:
 
 * Install the cert-manager of your choice
   if it is not already installed.
-* Modify the `Deployment` manifest of each KLT operator component.
-* Add the `Certificate` CRD for the cert-manager you are using.
-
-## Modify the KLT manifest
-
-You must modify the KLT manifest for each KLT operator component
-to make it aware of the cert-manager you are using.
-These instructions implement
-[cert-manager.io](https://cert-manager.io/);
-the process is similar for other cert-managers.
-
-To configure KLT to use your cert-manager,
-change the `Deployment` manifest of each KLT operator component
-and **replace** the following `volumes` definition
-
-   ```yaml
-   - emptyDir: {}
```

Comment:
so can we leave our empty dir unchanged in the deployments?

Reply:
yes, we do not need to modify anything here :)